app/controllers/users_controller.rb

```diff
@@ -213,6 +212,8 @@ def profile
           flash.now[:warning] = I18n.t('users_controller.user_has_been_moderated')
         end
       end
+      @count_ans = @profile_user.answers.count
+      @count_ques = @profile_user.questions.length
```

Comment:
You can use the instance variables defined here - https://github.com/publiclab/plots2/blob/master/app/controllers/users_controller.rb#L143-L188

directly in the view?

Reply:
Also, the current results are inconsistent - 

![Screenshot from 2019-03-31 16-26-14](https://user-images.githubusercontent.com/20878070/55288208-b74c1e00-53d1-11e9-90f7-79e89681b7be.png)